types/address.go

```diff
@@ -237,39 +249,30 @@ func (aa AccAddress) Bytes() []byte {
 	return aa
 }
 
-// AccAddress.String() is expensive and if unoptimized dominantly showed up in profiles,
-// yet has no mechanisms to trivially cache the result given that AccAddress is a []byte type.
-// With the string interning below, we are able to achieve zero cost allocations for string->[]byte
-// because the Go compiler recognizes the special case of map[string([]byte)] when used exactly
-// in that pattern. See https://github.com/cosmos/cosmos-sdk/issues/8693.
```

Comment:
Please keep part of this explanation along with the issue number, as it will serve folks in the future well.

Reply:
But the `map` part is not relevant any more here - we are not using map.

Reply:
Yeah so pick out the relevant parts and keep the issue reference too :-)

Reply:
That was done before submitting the PR

Reply:
I replied because of of your reply :-) in https://github.com/cosmos/cosmos-sdk/pull/8717#discussion_r584717908